### PR TITLE
Views

### PR DESCRIPTION
**Pull request by isaacpitwa**
- Implement the design from the sneak peek wireframes.
    - Styling is nice to have but focus on making sure everything functions first.
  - Use methods that you have created in the [Blog App - processing data in models](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Processing_data_model_project.md).

**NB: run `rake db:seed` to add dummy data**

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Isaac, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have a Pull Request description. Kindly ensure to add short summaries describing introduced changes in more detail of your Pull Request, and request for another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit.

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -3,5 +3,8 @@ def index
     @users = User.all
   end
```
Comment (mateo951) [inline app/controllers/users_controller.rb:4]: Users controller `#index` action renders all users to its corresponding Views index page 🔥

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -3,5 +3,8 @@ def index
...
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @most_recent_three_posts = @user.most_recent_three_posts
```
Comment (mateo951) [inline app/controllers/users_controller.rb:8]: Uses `#recent_post` method to render the user's 3 most recent post 🤓 Well done!

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,27 @@
...
+<%@posts.each do |post, idx|%>
+  <div class="user-post">
+    <h3><%=link_to post.title, user_post_path(@user, post) %></h3>
+    <p><%= post.text.slice(0...50) + '...' %></p>
+    <p class='count'>Comments <%=post.comments_counter%>, Likes <%=post.likes_counter%></p>
+    <div class="user-post-comments">
+      <h3>Comments</h3>
+      <%post.comments.each do |comment|%>
+        <p><%= comment.author.name %>: <%= comment.text %></p>
+      <%end%>
```
Comment (mateo951) [inline app/views/posts/index.html.erb]: Please be sure to use `post.recent_comments` to render 5 most recent comments in the `app/views/posts/index.html.erb` page as it is part of the requirements 📝 You already have the proper method to achieve this 🤓 The most_recent_five_comments on post model file 💪

**Review by mateo951 — CHANGES_REQUESTED** [review]
### Hi @isaacpitwa, 

**🔥Highlights🔥**
- App functionality 🐐

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @isaacpitwa 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

**Highlights**
- Well done making the required changes from the previous reviewer

Cheers and Happy coding!👏👏👏

Feel free to mention @MahmoudBakr23 to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**